frontend/ui/widget/button.lua

```diff
@@ -36,8 +36,8 @@ local Button = InputContainer:new{
     text = nil, -- mandatory
     text_func = nil,
     icon = nil,
-    icon_width = Screen:scaleBySize(DGENERIC_ICON_SIZE), -- our icons are square
-    icon_height = Screen:scaleBySize(DGENERIC_ICON_SIZE),
+    icon_width = Screen:scaleBySize(DGENERIC_ICON_SIZE * 0.8), -- our icons are square
+    icon_height = Screen:scaleBySize(DGENERIC_ICON_SIZE * 0.8),
```

Comment:
Might be worth a comment, if these 0.8 are really to match the size of "cfont" size 20 - and that all is worse if you change any of these 2 without tuning the other. Can you test that this is really the case ?

Reply:
Yes, the 0.8 factor corresponds only to cfont-20
https://github.com/koreader/koreader/blob/0283c44ff7b0904b02f29ab74574930c76f9d1a4/frontend/ui/widget/buttontable.lua#L25-L26
So changing buttons font face/size will cause the icon to look smaller/bigger than the text in other buttons.
Do you mean just adding a comment or to recalculate the factor depending on the font size of other text buttons in buttontable?

Reply:
No need to recalculate if we don't use a font other than the default anywhere (if someone needs another font size - and he shouldn't :) - he'll do the maths himself :)
Just a comment, may be move the text related settings together above the icons related settings, like this:
```lua
local Button = InputContainer:new{
    text = nil, -- mandatory (unless icon is provided)
    text_func = nil,
    text_font_face = "cfont",
    text_font_size = 20,
    text_font_bold = true,
    avoid_text_truncation = true,
    icon = nil,
    icon_width = Screen:scaleBySize(DGENERIC_ICON_SIZE * 0.8), -- our icons are square
    icon_height = Screen:scaleBySize(DGENERIC_ICON_SIZE * 0.8),
        -- These *0.8 make our icons nicely match the size of the default font
        -- above ("cfont", 20), when mixing both text and icon buttons.
    icon_rotation_angle = 0,
```